src/v2/components/UI/Box/index.ts

```diff
@@ -64,6 +85,30 @@ export const mixin = css`
   ${props => props.overflowScrolling && overflowScrolling}
 `
 
-export default styled.div`
+export interface BoxProps
+  extends React.HTMLProps<HTMLDivElement>,
+    DisplayProps,
+    PositionProps,
+    SpaceProps,
+    AlignItemsProps,
+    MinHeightProps,
+    MaxHeightProps,
+    MinWidthProps,
+    MaxWidthProps,
+    JustifyContentProps,
+    FlexDirectionProps,
+    BordersProps,
+    BorderColorProps,
+    BorderRadiusProps,
+    BgColorProps,
+    TextAlignProps,
+    FlexProps,
+    TopProps,
+    RightProps,
+    BottomProps,
+    LeftProps,
+    ZIndexProps {}
```

Comment:
👍

Reply:
Super minor, but this goes a long way for something that could potentially have many more additions: 

<img width="301" alt="Screen Shot 2019-05-07 at 10 30 10 AM" src="https://user-images.githubusercontent.com/236943/57320182-27298480-70b3-11e9-9916-d9b96bcaf6a4.png">

Reply:
yes plzz